recipes/bandit/all/test_package/conanfile.py

```diff
@@ -0,0 +1,29 @@
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps", "VirtualRunEnv"
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
+    def layout(self):
+        cmake_layout(self)
+
+    def generate(self):
```

Comment:
You can avoid it by declaring "CMakeToolchain" as generator, but using explicitly is more than welcome.

Reply:
Ok so I leave it like that ?

Reply:
@MartinDelille yes, you can leave generate with CMakeToolchain always.